sklearn/tree/tree.py

```diff
@@ -300,6 +300,10 @@ def fit(self, X, y, sample_weight=None, check_input=True,
         elif self.presort == 'auto':
             presort = True
 
+        if type(presort) is not bool:
```

Comment:
I think `not isinstance(presort, bool)` is more readable.

Reply:
Yes, thanks for the suggesstion